changelog/fix-ET-2184-breadcrumb-single-order-page

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: tweak
+
+Added Orders page breadcrumb for the Order Detail page
```

Comment:
This doesn't belong right @dpanta94 ? It is just part of req for the bucket which isn't released right?

Reply:
Yes, if you want for any reason the GH action to be passing, changelogger supports changelog enries of `patch` significance without an `message`. As a result the GH action would pass, but no changelog entry would be added.

Not including the file at all is also fine.